### DISCUSSION TITLE
spa middleware

### DISCUSSION THREAD

**Discussion by rhamadsani** [alosaur/alosaur#152] [Q&A]
Hi I'm new to deno & I love this Third Party Module,
does SPA stand for single page application? Sorry if the question is less interesting but I need it to build a SPA app.

**Comment by irustm** [comment]
Yes, you're right.

You can see an example [here](https://github.com/alosaur/alosaur/tree/master/examples/spa).

In the `www` folder in this case must be your SPA application

**Reply by rhamadsani** [reply]
ok thanks a lot buddy.
